lab_1/main.py

```diff
@@ -70,9 +67,8 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list or None:
         return None
     if freq_dict.keys() and top_n > 0:
         top_words = []
-        freq_words = sorted(freq_dict.items(), key=lambda x: x[1], reverse=True)
-        for element in freq_words[:top_n]:
-            top_words.append(element[0])
+        freq_words = sorted(freq_dict.keys(), key=freq_dict.get, reverse=True)
+        top_words.extend(freq_words[:top_n])
```

Comment:
why extend? just return

Reply:
I see your point now. The function is now 5 lines long